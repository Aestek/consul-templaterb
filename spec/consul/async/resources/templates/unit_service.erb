Service Consul
<%
  service('consul').each do |instance|
%><%= instance.service_address %>:<%= instance['Service']['Port'] %> (<%= instance['Node']['Node'] %>)
<%
  end
%>---
Service Consul filter UI
<%
  service('consul', tag: 'UI').each do |instance|
    # The mocks do not simulate queries
    if instance['Service']['Tags'].include? 'UI'
%><%= instance['Node']['Node'] %>:<%= instance['Service']['Port'] %>
<%
    end
  end
%>