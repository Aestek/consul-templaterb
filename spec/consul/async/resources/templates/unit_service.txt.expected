Service Consul
5.6.7.8:8300 (consul01-par.central.criteo.preprod)
10.251.1.96:8300 (consul02-par.central.criteo.preprod)
10.251.1.97:8300 (consul03-par.central.criteo.preprod)
10.251.1.99:8300 (consul04-par.central.criteo.preprod)
---
Service Consul filter UI
consul04-par.central.criteo.preprod:8300
